wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -1,6 +1,7 @@
 {% load i18n %}
 {% load l10n %}
 {% load wagtailadmin_tags %}
+<div class="w-sr-only">Press 'enter' to select an item, use up and down arrows to move the item, press 'enter' to deselect the item.</div>
 <table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}">
```

Comment:
```suggestion
<table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}" aria-descrption"Press 'enter' to select an item, use up and down arrows to move the item, press 'enter' to deselect the item.">
```

Remember to translate this also.